src/main/java/com/stripe/model/StripeError.java

```diff
@@ -205,4 +205,14 @@ public class StripeError extends StripeObject {
    */
   @SerializedName("type")
   String type;
+
+  @SerializedName("developer_message")
+  String developerMessage;
+
+  public String getMessage() {
```

Comment:
Does this override the getter for `message` property?

We might want to handle this logic closer to Exception creation. StripeError is more of a data class so it should reflect what we receive from the service.